butter_cms/__version__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.8'
```

Comment:
Since the version is now referenced in `setup.py` and the request header, I put it in this file. The ideal way to setup the `__version__` attribute would be in `__init__.py` per https://www.python.org/dev/peps/pep-0396/ but I've also seen plenty of packages put it in `__version__.py`. Doing this per the spec caused some import issues.

Reply:
Seems fine. Stripe puts there's into a `VERSION` file:  https://github.com/stripe/stripe-python/blob/master/VERSION